tests/helpers/__snapshots__/test_v1_to_v2.ambr

```diff
@@ -232,8 +232,8 @@
       meta:
         role: pkg_role
       identifier: <% fn.concat_ids('myapp', '_pkg_', fn.sanitize_id(fn.get_username('unknown_user')) | lower) %>
-      type: application package
       artifacts: []
+      type: application package
```

Comment:
these tests are sensitive to field ordering, which changed because of moving common fields to a base class. The output is functionally the same though.